### ISSUE TITLE
Request: Plots to spot bias (P-P, Q-Q and Zpred vs Zresid)

### ISSUE THREAD

**Issue by isakohrlund** [jasp-stats/jasp-desktop#2184]
Hi,

I'm trying to teach my students how to spot bias, but JASP currently lacks that functionality, so we're stuck with SPSS.

* Enhancement: P-P, Q-Q and zpred vs zresid plots
* Purpose: Spot bias

**Comment by EJWagenmakers** [comment]
Yes, cool; any specific analyses you have in mind? (regression?)

**Comment by isakohrlund** [comment]
Yes, well, that would be a start!
